components/nav.php

```diff
@@ -32,16 +32,6 @@
 				<li class="nav-item">
 					<a class="nav-link" href="/login">Volunteer Login</a>
 				</li>
-			<?php if ($USER->hasPermission('can_view_management_tab')): ?>
```

Comment:
@CollinsSpencer Did you ever get a chance to look into this on your Mac?

Reply:
I wasn't able to ever figure out why Mac didn't like it. I did just update to High Sierra and I can check if compatibility with that changed for some reason.

Reply:
Fixed in https://github.com/UNL-CSE-AMBASSADORS/VITA/pull/106